Lesson 7 homework/Lesson 7 homework/ViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ViewController.swift
+//  Lesson 7 homework
+//
+//  Created by Ernest Mihasenko on 22.03.22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        school.printInfo()
+    }
+}
+
+import UIKit
+import Foundation
+
+class School {
+    var students = [Student]()
+    
+    init(students: [Student] = [firstStudent, secondStudent, thirdStudent, fourthStudent]) {
+        self.students = students
+    }
+    func printInfo() {
+        for info in [firstStudent.printInfo(), secondStudent.printInfo(), thirdStudent.printInfo(), fourthStudent.printInfo()] {
+        print(info)
+        }
```

Comment:
You CAN'T access `firstStudent`, `secondStudent`, `thirdStudent`, `fourthStudent`. You should ONLY work with `students` property. 

You don't need to create a new array and definitely don't create an array with `printInfo()`. You need to call it inside the cycle on your `info` property